force-app/main/default/classes/DMLOperations.cls

```diff
@@ -31,13 +31,17 @@ public with sharing class DMLOperations {
      */
     public static Id insertNewAccount() {
         // Create a new Account instance
-
+        Account johnAccount = new Account();
         // Set some fields on the Account
-
+        johnAccount.Name = 'John';
```

Comment:
How could you randomize the name creation in the method so that anytime it ran it always generated a unique value?